activerecord/lib/active_record/log_subscriber.rb

```diff
@@ -67,7 +67,7 @@ def sql_color(sql)
       case sql
         when /\A\s*rollback/mi
           RED
-        when /\s*.*?select .*for update/mi, /\A\s*lock/mi
+        when /select .*for update/mi, /\A\s*lock/mi
```

Comment:
Could you update the pull request to removing it? If not it is fine. I can do myself.